lib/api/multiObjectDelete.js

```diff
@@ -483,6 +483,7 @@ function multiObjectDelete(authInfo, request, log, callback) {
         pushMetric('multiObjectDelete', log, {
             authInfo,
             bucket: bucketName,
+            keys: successfullyDeleted,
```

Comment:
it's ok if the array item is an object containing `{ versionId, key }` instead of being an array of strings?

Reply:
No it isn't. Is the ```successfullyDeleted``` array always composed of objects of this kind?

Reply:
yes, at least in any release with versioning implemented (>= 7.0)